.github/workflows/ci.yaml

```diff
@@ -783,6 +783,49 @@ jobs:
           name: pytest_buckets
           path: pytest_buckets.txt
           overwrite: true
+  audit-licenses:
+    name: Audit licenses
+    runs-on: ubuntu-22.04
+    needs:
+      - info
+      - base
+    if: |
+      needs.info.outputs.requirements == 'true'
+    steps:
+      - name: Check out code from GitHub
+        uses: actions/checkout@v4.1.7
+      - name: Set up Python ${{ env.DEFAULT_PYTHON }}
+        id: python
+        uses: actions/setup-python@v5.1.0
+        with:
+          python-version: ${{ env.DEFAULT_PYTHON }}
+          check-latest: true
+      - name: Restore full Python ${{ env.DEFAULT_PYTHON }} virtual environment
+        id: cache-venv
+        uses: actions/cache/restore@v4.0.2
+        with:
+          path: venv
+          fail-on-cache-miss: true
+          key: >-
+            ${{ runner.os }}-${{ steps.python.outputs.python-version }}-${{
+            needs.info.outputs.python_cache_key }}
+      - name: Install pip-licenses
+        run: |
+          . venv/bin/activate
+          pip install pip-licenses
```

Comment:
This doesn't pin a version for `pip-licenses`. It could create issues if a new (incompatible) version is released. Maybe it would be better to add `pip-licenses` to `requirements_test.txt` directly? That way it's also available in the local environment.